ChangeLog.md

```diff
@@ -4,21 +4,198 @@ Changelog
+- Tweaked ship design screen layout
+- Ship design screen description reworked to be toggled active and multiline
+- Improved objects list refresh responsiveness
+- Made objects list column default wider
+- Added outer border highlighting of GUI window with input focus
+- Added widget for bold font texture file in options window
+- Moved font options to separate tab
+- Added vertical scale labels to empire statistic graphs
+- Added a right-click popup menu to empire statistic graphs, with several display options
+- Made the window resize tab bigger
+- Set a minimum size for some windows to prevent them from being made too small to be usable
+- Reworked pedia window layout, putting search bar and navigation buttons at top, similar to a web browser standard layout
+- Fixed display of circle arrow icons on production screen
+- Made options.xml contain only modified options by default, but added a button to output the full options database to the options window
+- Reworked splash / menu screen layout
+- Added time to produce at the currently selected location with available PP to production item tooltips
+- Added an option to reset empire readiness when joining a game, so orders can be revised even after they were submitted
+- Replace total cost and minimum production time with turns to produce, if fully funded, at current production location to the producible items list
+- Added indication in tooltips when production cost or minimum time are location-dependent and no location is selected
+- Added option to disable disclosure of galaxy seed
+- Prohibit typing some characters into galaxy seed text box, to prevent pedia layout script injection
+- The server stop auto-advancing turns after a player has won the game
+- Relabelled Resign button to Exit to Menu
+- Removed unused / redundant keymaps configuration file
+- Repeatedly clicking to close the game window will override the default autosave and exit faster
+
+#### Content and Balance
+
+- New Stuff
+    - Added Arc Disruptor ship weapon part
+    - Added sitreps for gifting to allied empires
+
+- Reworked what weapons can target fighters, ships, and planets
+    - Mass drivers, Lasers, Plasmas, Death Rays, and Spinal antimatter cannon attack ships and planets (no fighters)
+    - Interceptors attack enemy fighters only
+    - Strike Fighters (formerly called just "fighters") attack ships and fighters
+    - Bombers attack ships only
+    - Heavy bombers are re-enabled and attack ships and planets
+
+- Rebalanced fighters
+    - Interceptors launch 50% faster than other fighters
+    - Fighter refinement techs increase interceptor capacity and launch rate, not damage
+    - Techs increase strike fighter damage by +2, bomber damage by +3, and heavy bomber damage by +6
+    - Adjusted costs and build times
+    - Species piloting traits give -1 to +3 for all fighters except interceptors
+
+- Ships
+    - Ship production cost increases ("upkeep") also count ships on the production queue
```

Comment:
`production queue` is ambiguous.  Do you mean ships in the production queue that already get any production points or all ships?

Reply:
added `that have received some production points`